### ISSUE TITLE
notation build failing

### ISSUE THREAD

**Issue by priteshbandi** [notaryproject/notation#247]
Looks like [notation-go/pull/64](https://github.com/notaryproject/notation-go/pull/64) commit broke notation builds. I’m getting following error:
```
github.com/notaryproject/notation/cmd/docker-notation imports
        github.com/notaryproject/notation-go/signature: module github.com/notaryproject/notation-go@latest found (v0.9.0-alpha.1), but does not contain package github.com/notaryproject/notation-go/signature
```
 Verified that notation-go does not contain package pkg/registry (edited) 

### Repo steps
1. Checkout latest version of [notation](https://github.com/notaryproject/notation) code.
2. Run ` go clean -modcache && go mod tidy`

**Comment by binbin-li** [comment]
I'll bump up the notation-go version and make the according change.

**Comment by binbin-li** [comment]
Closing it as it's resolved by #248
